docs/Quantization.md

```diff
@@ -186,3 +186,20 @@ allow the hardware to perform a simple comparison. By normalizing both sides of
 the 'max' operation to the same scale we enable this efficient optimization.
 
 For more specific graph optimizations check [here](Optimizations.md#quantization-specific-optimizations).
+
+## Row-wise Quantization
+
+Row-wise (or channel-wise) quantization is an important way to minimize accuracy drop.  
+Glow supports row-wise quantized FullyConnected ```RowwiseQuantizedFullyConnected)```
+which is enabled by an image-classifier/loader option "-enable-rowwise".
+
+For the regular quantized FC, we quantize the whole weights tensor with the same 
+scale and offset, which are computed based on the max and min of the entire tensor). 
+But for row-wise, after getting min_i and max_i for each row i, we compute the pair 
```

Comment:
Please highlight min_i and max_i and i, so it looks nicer. You can try using math notation (either $ or $$). If it doesn't work, use ``` code notation.